### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by jeanmichelcote** [garetmckinley/styled-babel-sublime#2]
Thanks a lot for the package, I know I will appreciate it.

But for the time being, It is not clear to me how should I install your fork since I already have the Babel package installed on my ST3.

Should I uninstall it completely and reinstall? Is merged to the official Babel package?

I tried just downloading your repo from the [main webpage](https://garetmckinley.github.io/styled-babel-sublime/) and swapping your files in the Babel package. Didn't work, nothing happened.

Any tips are welcome!

**Comment by ariamckinley** [comment]
Yeah, currently the master branch on this got rolled back to the original babel-sublime package. To get the styled-components changes, download [from here](https://github.com/garetmckinley/styled-babel-sublime/archive/babel-sublime.zip) instead. Hopefully the update for the official babel-sublime package will get released soon, so you can start using that.

**Comment by edheltzel** [comment]
Does this work? I can't get it installed...

**Comment by ariamckinley** [comment]
It's been merged into the official babel-sublime repo since february, can you please open an issue there and request a new version to be officially released? Otherwise you can install that repo from the source, this repo is deprecated
